src/ngraph/runtime/cpu/cpu_layout_descriptor.hpp

```diff
@@ -37,36 +36,36 @@ namespace ngraph
             class LayoutDescriptor : public ngraph::descriptor::layout::TensorViewLayout
             {
             public:
-                LayoutDescriptor(const ngraph::descriptor::TensorView& tv,
-                                 const AxisVector& tv_axis_order);
+                LayoutDescriptor(const ngraph::descriptor::TensorView& tv);
                 ~LayoutDescriptor() override {}
-                size_t get_size() override { return size; }
-                size_t get_offset() const { return offset; }
+                size_t get_size() override { return m_size; }
+                size_t get_offset() const { return m_offset; }
                 size_t get_index_offset(const std::vector<size_t>& indices) override;
 
-                const Strides& get_strides() const override { return strides; }
+                const Strides& get_strides() const override { return m_strides; }
+                void set_strides(Strides& strides) { m_strides = strides; }
                 bool operator==(const TensorViewLayout& other) const override;
 
-                void set_mkldnn_format(const mkldnn::memory::format& format)
+                const mkldnn::memory::desc& get_mkldnn_md() const { return m_mkldnn_md; }
+                void set_mkldnn_md(const mkldnn::memory::desc md) { m_mkldnn_md = md; }
```

Comment:
where is this getting called?

Reply:
in layout pass.

Reply:
ah ok, got it. github requires us to manually press the "Load Diff" if the file is too big.  Hence my Command + F couldn't locate it